packages/interface/src/capability.ts

```diff
@@ -436,6 +436,12 @@ export interface InvalidSignature extends Failure {
   readonly delegation: Delegation
 }
 
+export interface SessionEscalation extends Failure {
```

Comment:
can you include a comment or link to doc on what 'escalation' means in this context? e.g. if I am debugging an error and see its this one, I'd want to read that to figure out how to interpret the error and understand if there is a chance I can do something appropriate if I catch it

Reply:
after some reading, I think I grok that it's a pretty generic error and if someone encounters it, retrying the request wont help because there's something wrong in their authorization

Reply:
Yes it's error that occurs when invalid `ucan/attest` is found. Invocation with same proof chain would indeed produce same results.